TCAT/Cells/LargeDetailTableViewCell.swift

```diff
@@ -8,13 +8,20 @@
 
 import UIKit
 
+protocol LargeDetailTableViewDelegate: class {
+    func collapseCells(indexPath: IndexPath)
```

Comment:
instead of using `IndexPath`, it might be better to pass in the cell instead so that you can access the indexPath from the tableview ie. `func largeDetailTableViewCellDidCollapseCells(_ cell: LargeDetailTableViewCell)`